.github/workflows/rm_gh_cache_after_pr_closed.yml

```diff
@@ -0,0 +1,45 @@
+name: Cleanup caches after PR closed
```

Comment:
Let's rename the file with hyphens: `x-y-z.yml` (to be consistent with other files)

Reply:
Oh, yes.